lib/rubocop/cop/offense.rb

```diff
@@ -98,7 +98,7 @@ def initialize(severity, location, message, cop_name, # rubocop:disable Metrics/
       #
       # @return [Boolean]
       #   whether this offense can be automatically corrected via
-      #   autocorrect or a todo.
+      #   autocorrect. This includes todo comments, when requested.
```

Comment:
It would be better to add a bit more explanation about "when requested".

Reply:
👍 I had that in there initially but thought it was too much